python/pycylon/data/aggregates.pxd

```diff
@@ -24,7 +24,10 @@ cdef extern from "../../../cpp/src/cylon/compute/aggregate_kernels.hpp" namespac
         CCOUNT 'cylon::compute::COUNT'
         CMIN 'cylon::compute::MIN'
         CMAX 'cylon::compute::MAX'
-
+        CMEAN 'cylon::compute::MEAN'
+        CVAR 'cylon::compute::VAR'
+        CNUNIQUE 'cylon::compute::NUNIQUE'
+        CQUANTILE 'cylon::compute::QUANTILE'
```

Comment:
To give quantile we would have to expose the "compute::Op"s. Because we need to get that quantile argument. What shall we do there? @vibhatha

Reply:
Yes, I will work on another PR for this. I also thought about this.